ac

```
@@ -2033,6 +2085,48 @@ while true; do
             ensure_managed_venv_exists $1
             option_submitted="--venv-stop"
             ;;
+      --refresh-env|--refresh-env=?*)           # command 
+            option_submitted="--refresh-env"
```

All of this seems redundant, is better to keep it as the other options , only set option submitted and ensure that managed venv exists, when the option is called in the code below it should be something like 


```
venv_refresh $name $all
```

Then inside your code you check if $all is populated, take a look at this extract from `ac-test` function 
```
    mark=$9
    if [ "$mark" ]; then
        mark="-m ${mark}"
    fi
```

You can do something like this to move the logic of analyzing the parameters to the actual function and keep this part clean. Then inside the actual function you select what is going to be done, if option `all` is selected along `name` you can throw an error saying that it is not supported, if only `all` has a value then act on case `all` if not act on case `name` else act on case `*`.

---

resolved

---

This is not resolved, this portion of the code is still redundant, there is no need to have this logic here, please review `ac-test` option to see how you can handle multiple arguments inside the same function.